lms/djangoapps/grades/api/views.py

```diff
@@ -185,7 +185,7 @@ def get(self, request, course_id):
             return grade_user
 
         prep_course_for_grading(course, request)
-        course_grade = CourseGradeFactory().create(grade_user, course)
+        course_grade = CourseGradeFactory().read(grade_user, course)
```

Comment:
Wouldn't that be:
`course_grade.passed if course_grade else None` 
etc.? Or is my Python failing me? :)